.gitattributes

```diff
@@ -0,0 +1,8 @@
+/driver-testsuite export-ignore
```

Comment:
Well, the cache contains older archives. At one point, we were lucky to avoid hitting the rate limit.

I totally agree that a separate repo would be better for this particular reason. But it is not done yet, so excluding it from the archive today is wrong.